examples/rmg/commented/input.py

```diff
@@ -150,6 +150,16 @@
 )
 
 options(
+    	#provides a name for the seed mechanism produced at the end of an rmg run default is 'Seed'
+    name='RMGButane',	
+	#if True every iteration it saves the current model as a libraries/seeds
+	#(and deletes the old one)
+	#Unlike HTML this is inexpensive time-wise
+	#note this one will be generated at the end of a completed run and some incomplete runs even
+	#if False
+    generateSeeds=True,
+	#If True the mechanism will also be saved directly as seeds/libraries in the database
```

Comment:
perhaps consider:
"#If True, the final seed mechanism will also be saved directly in the database as thermo and kinetic libraries"